addons/binding/org.openhab.binding.onebusaway/ESH-INF/thing/api.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onebusaway"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="api">
+        <label>OneBusAway API</label>
```

Comment:
It's literally an API key, and each server admin (for different transit regions) has different policies on how to hand these out, as far as I can tell.  In the pugent sound region (Seattle, WA), you just email them and they give you an API key.